add_nyt_review_link.py

```diff
@@ -162,13 +170,27 @@ def run(self) -> None:  # overwrites the AbstractBotJob run method
         comment = 'Add NYT review links'
         with open(self.args.file, 'r') as fin:
             review_record_array = json.load(fin)
-            for review_record in review_record_array:
-                self.__process_review_record(review_record,
-                                             comment, job_results)
+            last_shown_on_progress_bar = 0
+            progress_bar_capacity = len(review_record_array)
+            with tqdm(total=progress_bar_capacity, unit='reviews') as pbar:
+                for index, review_record in enumerate(review_record_array):
+                    self.__process_review_record(review_record,
+                                                 comment, job_results)
+                    if index > 0 and index % 25 == 0:
+                        last_shown_on_progress_bar = index
+                        pbar.update(25)
+                pbar.update(progress_bar_capacity - last_shown_on_progress_bar)
```

Comment:
why manually control tqdm vs use the wrapper around the list?

Reply:
It doesn't look good, the progress bar printed under every isbn and just makes reading logs harder. The tags bot prints the progress line after every bestseller group, so the progress bar is shown in the logs every 10 ISBNs or so.  That code mimics the same output.